### DISCUSSION TITLE
Suggestion of improving the "Fee system"

### DISCUSSION THREAD

**Discussion by datorymen** [sunflower-land/sunflower-land#1333] [Ideas]
I have been playing Sunflower-Land since end May. I like the project and would like to contribute. So I tried to learn Aseprite and submitted two art works already. 

But now I am a bit concerned about the “Fee system”. I worry it will decrease the passion of players.

So what fees I encountered so far?

1.	"Sync fee" – This is well-known already. We know it costs USD0.1 and have to sync when we have no stock of seeds or axes.
2.	“Have to sync fee” – I met this fee when I collected eggs, cut trees or collected crops. I know its intention is to help us keep resources safe but isn’t it players’ own choice to do so? I just feel we “have to” do so otherwise we can not continue to play.
3.	“Trading fee” – It is fixed 0.25MTIC.  When MATIC was USD0.3, it is not a big amount. But when MATIC is USD0.9, it is not a small amount if you want to do more trading.  

Suggestion:  
I think an active trading system would help players find more interest to keep playing. 

Can I suggest lowering the Trading fee to be same as the sync fee USD0.1 and also let it sync for us each time we list an item? This also means we need to increase the trading limits or remove it. That will encourage people to trade more when they are out of seeds or axes, then the whole market becomes active.

Please add on if there’s another fee I don’t know yet.  I also hope the project team could earn more money so you can try this way for 1-2 month and see whether we will get more “volunteer sync” than the original “have to sync”.

I love Sunflower-Land. I wish the game will become the No. 1 Web3 game!

**Comment by LittleEins** [comment]
Maybe you want to read this discussion:
https://github.com/sunflower-land/sunflower-land/discussions/1326
If this were to be implemented players will find goals that they won't even have to think about the fees.

**Comment by DemonDG** [comment]
I have two T3 now.I actively want to get the next one.But sync fee make me poor.
